pontoon/base/templatetags/helpers.py

```diff
@@ -1,8 +1,10 @@
 import cgi
 import datetime
 import json
-import urllib
-import urlparse
+from six.moves.urllib.parse import urlparse, parse_qsl
+from six.moves.urllib.parse import quote_plus, ParseResult
+from six.moves.urllib.parse import urlencode as urllib_urlencode
+from six import text_type
```

Comment:
Please rewrite to: 

```python
from six import text_type
from six.moves.urllib import parse
```

And then use ``parse.xxx`` to replace the various ``urlparse.xxx`` and ``urllib.xxx``. It is generally a better practice to know what package a method comes from, and numerous imports like this should be avoided. The cost of importing the entire module instead is very low.

Note: you can rename ``parse`` to something better if you want. I admit that it is not a very good name and it makes readability harder. Maybe ``from six.moves.urllib import parse as six_parse`` is better?